handler/areas.go

```diff
@@ -108,6 +108,84 @@ func (h *Areas) Get(w http.ResponseWriter, r *http.Request) {
 	h.respond.JSON(ctx, w, http.StatusOK, resp)
 }
 
+// GetID returns the information for a particular area
+// GET /population-types/{population-type}/
+func (h *Areas) GetID(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+
+	cReq := cantabular.GetAreasRequest{
+		PaginationParams: cantabular.PaginationParams{
+			Limit:  1,
+			Offset: 0,
+		},
+		Dataset:  chi.URLParam(r, "population-type"),
+		Variable: chi.URLParam(r, "area-type"),
+		Category: chi.URLParam(r, "area-id"),
+	}
+
+	logData := log.Data{
+		"population_type": cReq.Dataset,
+		"area_type":       cReq.Variable,
+		"query":           cReq.Category,
+	}
+
+	// only return results for published population-types on web
+	if !h.cfg.EnablePrivateEndpoints {
+		if err := h.published(ctx, cReq.Dataset); err != nil {
+			h.respond.Error(ctx, w, http.StatusNotFound, &Error{
+				err:     errors.Wrap(err, "failed to check published state"),
+				message: "population type not found",
+				logData: logData,
+			})
+			return
+		}
+	}
+
+	res, err := h.ctblr.GetAreas(ctx, cReq)
```

Comment:
I don't think you should be using the existing GetAreas func as it will use the provided category as a search, not an exact match. We should be writing a new Cantabular query which only returns the exact area provided.

Reply:
the [ticket](https://trello.com/c/hcic5vpQ/5795-create-get-population-types-population-type-area-types-area-type-areas-area-id-endpoint) specifically tells me to use it. Happy to explore your solution if there is agreement within the wider team that current description of the ticket is bad practice